packages/verticaltabs/src/react/index.tsx

```diff
@@ -49,7 +46,11 @@ const VerticalTabs = React.forwardRef((props, ref) => {
 
   return (
     <Context.Provider value={contextValue}>
-      <ul {...styles.verticaltabs()} ref={ref} {...rest}>
+      <ul
+        className={classNames('psds-verticaltabs', className)}
```

Comment:
Your commit message says that there's a breaking change, moving the className to parent  node. I don't think that's the case. It was on the parent node already. This seems to also be true from the other subcomponents in this package that I observed. This is probably true for all 3 commits in this PR.

Reply:
Copy paste error looking for what was a good default message for this rather repetitive tasks